### PR TITLE
add ExchangeTokenAccount. add ExchangeToken example. Format examples.

### PR DESCRIPTION
**Pull request by soroushjp**

### CODE REVIEW

File: plaid/exchange-token.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 
 // ExchangeToken (POST /exchange_token) exchanges a public token for an access token.
 //
-// See https://github.com/plaid/project-csa
+// See https://github.com/plaid/link
```
Comment (michaelckelly) [inline plaid/exchange-token.go]: :+1:

File: plaid/plaid.go [hunk 2]
```diff
@@ -131,7 +131,9 @@ type mfaResponse struct {
 type postResponse struct {
 	// Normal response fields
 	AccessToken  string        `json:"access_token"`
+	AccountId    string        `json:"account_id"`
 	Accounts     []Account     `json:"accounts"`
+	AccountToken string        `json:"stripe_bank_account_token"`
```
Comment (michaelckelly) [inline plaid/plaid.go]: Maybe `BankAccountToken`?

**Comment by michaelckelly** [comment]
LGTM other than that comment - thank you @soroushjp.

**Comment by michaelckelly** [comment]
Also closes https://github.com/plaid/plaid-go/issues/14!

**Comment by soroushjp** [comment]
> Did we mean to include the file example?

I thought we had this included in the repo previously (which I thought was strange), removing now.

**Comment by soroushjp** [comment]
:zap:

**Comment by michaelckelly** [comment]
:turtle:

**Pull request merged**